src/i18n.js

```diff
@@ -223,6 +223,33 @@ ${map(pot.items, (item) => {
   });
 }
 
+/**
+ * Sync po by the pot file and apply defaults
+ * @function syncENPoFileWithDefaults
+ * @return {undefined}
+ */
+function syncENPoFileWithDefaults() {
+  const filename = 'locales/en/LC_MESSAGES/volto.po';
+  const po = Pofile.parse(fs.readFileSync(filename, 'utf8'));
+  const messages = getMessages();
+  fs.writeFileSync(
+    filename,
+    `${formatHeader(po.comments, po.headers)}
+${map(keys(getMessages()).sort(), (key) => {
+  const poItem = find(po.items, { msgid: key });
+  return [
+    `${map(poItem.references, (ref) => `#: ${ref}`).join('\n')}`,
+    `msgid "${poItem.msgid}"`,
+    `msgstr "${
+      poItem?.msgstr?.length && poItem?.msgstr[0] !== ''
+        ? poItem.msgstr[0]
```

Comment:
If I understand the code well, you don't overwrite the English translation? I think we should take the Volto default messages as the canonical and always overwrite. I understand that there's a use case where somebody might want to overwrite the default messages. In that case a custom integration/workflow should be implemented (by those that want it).